code/parachain/frame/assets-registry/src/benchmarking.rs

```diff
@@ -82,11 +83,11 @@ benchmarks! {
 		.expect("Asset details are non-duplicate and valid");
 
 		let asset_info_update = AssetInfoUpdate {
-			name: Some(Some(BiBoundedAssetName::from_vec(b"Cooler Kusama".to_vec()).expect("String is within bounds"))),
-			symbol: Some(Some(BiBoundedAssetSymbol::from_vec(b"CKSM".to_vec()).expect("String is within bounds"))),
-			decimals: Some(12),
-			existential_deposit: Some(T::Balance::from(0)),
-			ratio: Some(None),
+			name: UpdateValue::Set(Some(BiBoundedAssetName::from_vec(b"Cooler Kusama".to_vec()).expect("String is within bounds"))),
+			symbol: UpdateValue::Set(Some(BiBoundedAssetSymbol::from_vec(b"CKSM".to_vec()).expect("String is within bounds"))),
+			decimals: UpdateValue::Set(Some(12)),
+			existential_deposit: UpdateValue::Set(T::Balance::from(0)),
```

Comment:
could this just be `T::Balance::zero()`?

Reply:
:sweat_smile: